cloud/docker/docker.py

```diff
@@ -320,6 +320,12 @@
     default: false
     aliases: []
     version_added: "2.0"
+  stop_timeout:
+    description:
+      - Set the timeout for docker stop commands before SIGKILL is sent.
```

Comment:
I think it'd be friendlier to explicitly specify which actual states this is relevant for, to take out a bit of guesswork. From the source, it looks like `state: stopped` and `state: restarted`, correct?